source/extensions/filters/http/jwt_authn/response_data.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstddef>
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+/**
+ * Struct to keep track of verifier completed and responded state for a request.
+ */
+struct ResponseData {
+  // if verifier node has responded to a request or not.
+  bool has_responded_;
```

Comment:
you need to initialize these members especially since the default constructor is used by the _map.

Reply:
done